docs/output.md

```diff
@@ -0,0 +1,20 @@
+# Arcadia-Science/metagenomics Output
+
+This page describes the output files and report produced by the pipeline. The directories listed below are created in the results directory after the pipeline has finished.
+**This page is an active work in progress as the workflow is in development**
+
+## Pipeline Overview
+
+This pipeline takes in metagenomic reads, performs QC, assembly, and subsequent mapping of reads back to the resulting assemblies. Steps downstream of assembly and mapping are identical for processing data from both sequencing types.
+
+### Illumina-Specific Outputs
+
+Read QC and adapter trimming is performed with `fastp`, and assembly performed with `metaspades`. Reads are mapped back to the assembly with `bowtie2`.
```

Comment:
These sections don't really talk about the outputs but more so about the tools used. Will you add those details later?

Reply:
yes, since I'm still actively working on additions I will add the information about outputs/report results later